src/components/SourceFooter.js

```diff
@@ -13,10 +13,10 @@ const { shouldShowFooter, shouldShowPrettyPrint } = require("../utils/editor");
 require("./SourceFooter.css");
 
 function debugBtn(onClick, type, className = "active", tooltip) {
-  className = `${type} ${className}`;
-  return dom.span(
+  className = `${type} ${className} action`;
+  return dom.button(
```

Comment:
I'll create a new issue for following up on our other SVG buttons so that they can as accessible